core/runtime/src/main/java/org/eclipse/sw360/antenna/util/HttpHelper.java

```diff
@@ -54,7 +54,7 @@ public File downloadFile(String url, Path targetDirectory, String filename) thro
             HttpEntity entity = response.getEntity();
             if (entity != null) {
                 try (InputStream is = entity.getContent()) {
-                    FileUtils.copyInputStreamToFile(is, targetFile);
+                    Files.copy(is, targetFile.toPath());
                 }
```

Comment:
We have to be careful here because the version from FileUtils does some more things under the hood. It can automatically create missing directories and overrides an existing file. This not handled by Files.copy(). I assume that is the reason why tests are failing in the PR build.

Reply:
Thanks for pointing this out! I'll check...

Reply:
Actually, the reason for the test to fail is that the [file already exists](https://travis-ci.com/github/eclipse/antenna/jobs/296887076#L6856). I'm wondering whether this function *should* indeed overwrite existing files, or whether we should rather cleanup the file in the test. Any opinion?

Reply:
I decided to mimic the original behavior to overwrite the file for now.